installer_scripts/install-yb-voyager-debezium

```diff
@@ -8,8 +8,9 @@ trap on_exit EXIT
 LOG_FILE=/tmp/install-yb-voyager-debezium.log
 YUM="sudo yum install -y -q"
 
+# Remember to update the S3 repo with the correct version of pre-biult debezium-server, if any change in these values.
 DEBEZIUM_VERSION="2.2.0-SNAPSHOT"
-DEBEZIUM_COMMIT_HASH="7164f6280183d7d7c92088537792e9200defad09"
+DEBEZIUM_COMMIT_HASH="4d513804b713505a4e5c8479d10c92f047f2a253"
```

Comment:
@rahulb-yb please update it with the latest commit hash
`4c892e5e3d5b4fad26daa4bf6e253cff0eb454a7`